src/App.jsx

```diff
@@ -12,34 +14,22 @@ export const misha = {
 export const olya = {
   name: 'Olya',
   sex: 'f',
+  age: 25,
```

Comment:
```suggestion

```

Reply:
Consider keeping the task data without a change and applying the conditional rendering for this field